tests/unittest/_utils/test_anti_zombie.py

```
@@ -0,0 +1,141 @@
+print("CHILD_READY", flush=True)
+time.sleep(300)
+'''], stdout=subprocess.PIPE, text=True)
+# Relay the child's readiness + pid to our stdout.
+child.stdout.readline()  # wait for CHILD_READY
+print(f"CHILD_PID={child.pid}", flush=True)
+time.sleep(300)
+"""
+
+
+def test_prctl_kills_child_when_parent_dies():
+    parent = subprocess.Popen(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)